packages/react-notifications/lib/react-notifications.js

```diff
@@ -16,6 +16,7 @@ export class Notifications extends Component {
 		this.setState({
 			hide: true
 		});
+		this.props.cbFunction ? this.props.cbFunction() : '';
```

Comment:
@creador-dev, I don't see how this callback function will work. Did you test it?